jaeger/spans.go

```diff
@@ -6,47 +6,32 @@ import (
 	"net/url"
 	"path"
 	"sort"
-	"strconv"
 	"time"
 
 	jaegerModels "github.com/jaegertracing/jaeger/model/json"
 
-	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/log"
+	"github.com/kiali/kiali/models"
 )
 
-// TODO / Question: make limit configurable? Selected from UI?
-const tracesLimit = 100
-
```

Comment:
This was for the first check that we do when the user go to the service details. We fetch the last 100 traces to check if there is an error in the tab (https://github.com/kiali/kiali-ui/blob/master/src/pages/ServiceDetails/ServiceDetailsPage.tsx#L121) where we fetch https://github.com/kiali/kiali/blob/master/jaeger/traces.go#L48 . The point is taht I forgot remove this because we fetch by tag error :( So good one

Reply:
no, it's my old code :) i was using it for spans